src/traffic_server/traffic_server.cc

```diff
@@ -1969,7 +1969,9 @@ main(int /* argc ATS_UNUSED */, const char **argv)
         start_HttpProxyServer(); // PORTS_READY_HOOK called from in here
       }
     }
+#ifdef OLD
```

Comment:
Should we keep this?

Reply:
Nope, missed that one.  Will take it out.